Test/e2e/Dockerfile.playwright

```diff
@@ -0,0 +1,36 @@
+# Use Ubuntu with .NET SDK
+FROM mcr.microsoft.com/dotnet/sdk:9.0
```

Comment:
I checked and saw the official docs say that we must use the official docker image that matches the version of playwright we are using in our tests. You absolutely must use mcr.microsoft.com/playwright:v1.55.0-noble as the base image. This is non-negotiable, do not try and attempt to work around it.